weave/trace_server/trace_server_interface.py

```diff
@@ -292,6 +292,20 @@ class TableCreateReq(BaseModel):
     table: TableSchemaForInsert
 
 
+class TableUpdateReq(BaseModel):
```

Comment:
I'm sure you put a lot of thought here and don't want to discount it, but did you consider an interface where instead of separate lists of inserts, deletes and appends, there's a single list and each element is an insert, delete or append?  this clears up a great deal of ambiguity, and more akin to what i've encountered in the past.  cf. bigtable